spec/requests/posts_spec.rb

```diff
@@ -0,0 +1,23 @@
+require 'rails_helper'
+
+RSpec.describe 'Posts', type: :request do
+  describe 'GET /users/:id/posts' do
+    it 'renders the index action with index view' do
+      get '/users/1/posts'
+      expect(response).to have_http_status(:success)
+      expect(response.status).to eq(200)
+      expect(response).to render_template('posts/index')
+      expect(response.body).to include('Here is a list of posts for a given user')
+    end
+  end
+
+  describe 'GET /users/:id/posts/:id' do
```

Comment:
#### OPTIONAL

- [ ] In your describe blocks, it would be better to use 'GET /index' or 'GET /show'  to describe the tests you are writing 👍🏿 

```
describe 'GET /index' do
  ...
end

describe 'GET /show' do
  ...
end
````